src/why_sponsor.njk

```diff
@@ -0,0 +1,110 @@
+            </div>
+        </div>
+        <div class="relative pl-12 md:pl-8 lg:pl-16 md:mt-0 z-10">
+            <img src="{{ env.baseUrl }}img/assets/sponsors.svg" alt="Sponsors" class="w-[80%] md:w-[90%] lg:w-[100%] block">
+            <img src="{{ env.baseUrl }}img/assets/star-pink-vector.svg" alt="Groovy star" class="absolute right-32 md:right-16 top-[14rem] md:top-[18.5rem] w-[10%] md:w-[25%] lg:w-[15%] animate-bounce-slow">
+            <img src="{{ env.baseUrl }}img/assets/circle-lavender.svg" alt="Circle Lavender" class="hidden md:block absolute right-[8rem] md:right-[14rem] top-[18rem] md:top-[29rem] w-[15%] md:w-[20%] lg:w-auto animate-bounce-fast">
+            <img src="{{ env.baseUrl }}img/assets/tetris.svg" alt="Tetris" class="hidden md:block absolute w-16 md:w-20 lg:w-20 right-16 md:right-32 bottom-[8rem] md:bottom-[18rem] rotate-12 animate-bounce-slower">
+            <img src="{{ env.baseUrl }}img/assets/ellipse-lime.svg" alt="Ellipse Lime" class="hidden md:block absolute w-[12%] md:w-[15%] lg:w-auto right-0 bottom-[16rem] md:bottom-[25rem] animate-bounce-slowest">
+            <img src="{{ env.baseUrl }}img/assets/circle-lavender.svg" alt="Circle Lavender" class="hidden md:block absolute w-[10%] md:w-[12%] lg:w-auto right-0 bottom-[12rem] md:bottom-[18rem] animate-bounce-slow">
+        </div>
+    </div>
+</div>
+<div class="relative">
+    <div class="absolute left-[10%] right-[10%] pt-10 md:-top-40 z-10 flex flex-col lg:flex-row items-start">
+        <div class="text-md lg:text-xl font-light flex-1">
+            {% set sponsorBenefits = [
+                {
+                    title: "Reach",
+                    description: "Access to 1,500+ attendees interested in your products and services and generate qualified leads."
+                },
+                {
+                    title: "Brand strength", 
+                    description: "Being part of the biggest and most prestigious Python conference in the world and supporting the nonprofit organization behind the Python language."
+                },
+                {
+                    title: "Connections",
+                    description: "Networking with attendees in person to create connections and provide detailed information about your products and services."
+                },
+                {
+                    title: "Recruiting",
+                    description: "Access to qualified job candidates. If you're hiring, there's no better place to find Python developers than PyCon India."
+                },
+                {
+                    title: "12 months of benefits",
+                    description: "Reach the Python community during PyCon India and beyond, with options for recognition on <a href=\"https://python.org\" class=\"text-blue-600 underline\">Python.org</a>, <a href=\"https://pypi.org\" class=\"text-blue-600 underline\">PyPI.org</a>, and more."
+                }
+            ] %}
```

Comment:
Consider taking the points from https://docs.google.com/document/d/1ASc9gjjTa8EtrYxT_yuq7m76RaTFNuSH6k0u_Hmt4Hc/edit?tab=t.0 and merging them together however logically they sound fit.

Reply:
Done